config.example.toml

```diff
@@ -942,3 +942,6 @@
 # Copy the linker, DLLs, and various libraries from MinGW into the Rust toolchain.
 # Only applies when the host or target is pc-windows-gnu.
 #include-mingw-linker = true
+
+# Whether to vendor dependencies for the dist tarball.
+#vendor = if "is a tarball source" || "is a git repository" { true } else { false }
```

Comment:
It shouldn't be necessary to vendor when building from a source tarball. We can just reuse the existing vendor directory.

Reply:
I thought about that, but then decided to leave the task to `cargo` instead of handling it manually. I couldn't get rid of the paranoia about users potentially cluttering the vendor folder.

Reply:
> It shouldn't be necessary to vendor when building from a source tarball. We can just reuse the existing vendor directory.

Exactly!

I'm building using a source release tarball (when packaging for pkgsrc), and expect the tarball to be self-contained.  Ref. issue #130708.